src/com/epam/homework/lists/StudentInitializer.java

```diff
@@ -0,0 +1,48 @@
+package com.epam.homework.lists;
+
+public class StudentInitializer {
+    private Student[] students = {
+            new Student("John", "Doe", 16),
+            new Student("Alexander", "Gates", 18),
+            new Student("Simon", "Miller", 20),
+            new Student("Alexander", "Anderson", 17),
+            new Student("Tim", "Jameson", 19),
+            new Student("Simon", "Whitaker", 18),
+            new Student("Andrew", "Johnson", 17),
+    };
+
+    public StudentInitializer() {
+    }
+
+    public StudentInitializer(Student[] students) {
+        if (students != null) {
+            this.students = students;
+        }
+    }
+
+    public Student[] initStudentArray() {
```

Comment:
It returns a clone of array hardcoded in StudentInitializer class. That array is used for testing to initialize  Student custom lists implemented in homework. I decided to get that array via this method because it returns the clone of the array and the underlying array can't  be mutated when we use it for different sorting and other operations.